tests/conftest.py

```diff
@@ -115,4 +115,16 @@ def _create_pipeline_instance_id(self) -> str:
     # disable httpx request logging (too verbose when testing qdrant)
     logging.getLogger("httpx").setLevel("WARNING")
 
+    # reset airflow db
+    from airflow.utils import db
+
+    for log in ["airflow"]:
+        logging.getLogger(log).setLevel("ERROR")
+
+    try:
+        db.resetdb()
```

Comment:
could you take a look at the function below in `tests/helpers/airflow_tests/utils.py`
```
def setup_airflow() -> None:
```
which we use to initialize airflow for our tests. it for example disables loading examples. but maybe it is irrelevant for the simple reset. is there anything we should also do here?